src/FontMetrics.php

```diff
@@ -130,14 +132,16 @@ public function loadFontFamilies()
         if (!defined("DOMPDF_FONT_DIR")) { define("DOMPDF_FONT_DIR", $fontDir); }
 
         $file = $rootDir . "/lib/fonts/dompdf_font_family_cache.dist.php";
-        $distFonts = require $file;
+        $distFontsClosure = require $file;
+        $distFonts = $distFontsClosure($rootDir);
 
         if (!is_readable($this->getCacheFile())) {
             $this->fontLookup = $distFonts;
             return;
         }
 
-        $cacheData = require $this->getCacheFile();
+        $cacheDataClosure = require $this->getCacheFile();
+        $cacheData = $cacheDataClosure($fontDir);
```

Comment:
Do I need to worry about the possibility of the cache file being from older dompdf? It could be fixed like this:

```suggestion
        $cacheDataClosure = require $this->getCacheFile();
        $cacheData = is_array(cacheDataClosure) ? $cacheDataClosure :  $cacheDataClosure($fontDir);
```

Reply:
Just want to check this comment and see what happens. Did you test the potential of running before and after the code is modified?

Reply:
no, my app isn't using this part of the code

Reply:
We would definitely need to add this logic to handle the alternate format.

Reply:
Alright, I added the condition.